api/rql/internal/primary/atime_test.go

```diff
@@ -0,0 +1,86 @@
+package primary
+
+import (
+	"testing"
+	"time"
+
+	"github.com/puppetlabs/wash/api/rql"
+	"github.com/puppetlabs/wash/api/rql/ast/asttest"
+	"github.com/puppetlabs/wash/api/rql/internal/predicate"
+	"github.com/puppetlabs/wash/api/rql/internal/predicate/expression"
+	"github.com/stretchr/testify/suite"
+)
+
+type AtimeTestSuite struct {
+	asttest.Suite
+}
+
+func (s *AtimeTestSuite) TestMarshal() {
```

Comment:
Could be worth a mini test suite for atime/ctime/mtime/crtime to avoid duplicating all the same testing.

Reply:
You could probably make it a test suite that you instantiate with the particular primary name and type.

Reply:
Yeah, I think that's a good idea. I'll go ahead and do that.

Reply:
Updated